src/contact/parallel/bvh_contact_manager.cc

```diff
@@ -58,6 +58,20 @@ namespace nimble {
     } );
 
     m_world.finish_iteration();
+
+    // Update contact entities
+    for ( auto &&face : contact_faces_ )
+    {
+      face.set_contact_status( std::count_if( m_last_results.begin(), m_last_results.end(),
+                          [&face]( NarrowphaseResult &res ){ return res.second_global_id == face.contact_entity_global_id(); } ) > 0 );
+    }
+
+    for ( auto &&node : contact_nodes_ )
+    {
+      node.set_contact_status( std::count_if( m_last_results.begin(), m_last_results.end(),
+                                              [&node]( NarrowphaseResult &res ){ return res.first_global_id == node.contact_entity_global_id(); } ) > 0 );
+    }
+
```

Comment:
@nmm0 
Is the purpose of these 2 loops to extract the faces and nodes in contact?
So the quantities calculated by ArborX in the pair (indices, offset)?

Reply:
Almost; contact_status_ is just a boolean used for debugging so it doesn't currently represent what other elements it is in collision with. It just states that the particular ContactEntity is involved in a collision. I don't think there is a good way to represent in the exodus file what pairs of collisions there are

Reply:
What about use the face id for the node "status"? 0 for no collision. The compliment doesn't work i.e. faces can have many node contacts. Here we could count the number of nodes that contact the face but this info is superfluous